### PR TITLE
Update app to 4.0.0

### PR DESCRIPTION
**Pull request by Aeris1One**
## Problem
- *The 3.9.0 is broken*

## Solution
- *Update to 4.0.0 (last version which supports php7.0)*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** : 
- [ ] **Approval (LGTM)** : 
- **CI succeeded** : 
[Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/ampache_ynh%20PR54/)  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by Aeris1One** [comment]
🤔
I used the *-all.tar.gz archive, which include dependencies, so no need to use composer to install them.

File: conf/app.src [hunk 1]
```diff
@@ -1,6 +1,6 @@
-SOURCE_URL=https://github.com/ampache/ampache/archive/3.9.0.tar.gz
-SOURCE_SUM=1e523b52c6402693102dac183d7597d6e0aebf63580167f45661d0f15f57c072
-SOURCE_SUM_PRG=sha256sum
+SOURCE_URL=https://github.com/ampache/ampache/releases/download/4.0.0/ampache-4.0.0_all.zip
+SOURCE_SUM=0f1f8a3c6b5c9511294a0db6d00efa5e
+SOURCE_SUM_PRG=md5sum
 SOURCE_FORMAT=tar.gz
```
Comment (maniackcrudelis) [inline conf/app.src:4]: If you use a zip instead of a tar.gz, change the format at this line.

**Comment by Aeris1One** [comment]
Finally upgraded to 4.1.1

**Comment by Aeris1One** [comment]
🤔 https://paste.yunohost.org/ucalohetuk

**Comment by maniackcrudelis** [comment]
> WARNING - cp: cannot create regular file '/var/www/ampache/config/ampache.cfg.php': No such file or directory

**Comment by Aeris1One** [comment]
Idk why, that appears even with the 3.9.0 -_-

**Comment by HugoPoi** [comment]
@Aeris1One which version of Yunohost do you use for your test ?

I pretty sure there is something wrong with `ynh_setup_source` inside Yunohost

**Comment by HugoPoi** [comment]
And we neep php 7.1 at minimum, so we need to use the new 3.8 feature for that see https://github.com/YunoHost/yunohost/pull/881

**Comment by Aeris1One** [comment]
@HugoPoi There's experimental helpers, we do not need to wait for 3.8, we can use experimental helpers.
I'll edit the app.src, but I don't have enough time to debug the install script... Feel free to create a PR on the top of this one

**Comment by maniackcrudelis** [comment]
Please wait until 3.8, would be a lot safer than to use the experimental helpers.
That php thing is quite tricky, it can explode at your face at any moment.

Even though, I do understand it can be frustrating

**Comment by HugoPoi** [comment]
I'm on it :sunglasses:

**Pull request closed without merging**